rotkehlchen/tests/fixtures/db.py

```diff
@@ -190,7 +191,7 @@ def database(
 ) -> Optional[DBHandler]:
     if not start_with_logged_in_user:
         return None
-
+    add_unique_cache_table(globaldb)
```

Comment:
Hmmm this line makes no sense since:
1. This is a user (encrypted) database fixture that has nothing to do with the global db
2. We should not populate tables dynamically. The test databases (`rotkehlchen/data/global.db` is the main one) should be upgrade and should contain the new table